conda_libmamba_solver/solver.py

```diff
@@ -757,13 +757,13 @@ def _notify_conda_outdated(
             return
 
         channel_name = current_conda_prefix_rec.channel.canonical_name
-        if channel_name == UNKNOWN_CHANNEL:
+        if channel_name in (UNKNOWN_CHANNEL, "@", "<develop>", "pypi"):
```

Comment:
Fixes a query issue when we are checking whether a conda update is available, but its source channel is "develop" because it was installed in dev mode. I added other commonly used fake channels just in case.

Reply:
Will this @ here stumble over authentication passed in?